Sources/Swift/Integrations/Performance/IO/Data+SentryTracing.swift

```diff
@@ -18,9 +18,9 @@ public extension Data {
     ///   - url: The location on disk of the data to read.
     ///   - options: The mask specifying the options to use when reading the data. For more information, see ``NSData.ReadingOptions``.
     /// - Note: See ``Data.init(contentsOf:options:)`` for more information.
-    init(contentsOfWithSentryTracing url: URL, options: Data.ReadingOptions = []) throws {
+    init(contentsOfWithSentryTracing url: URL, options: NSData.ReadingOptions = []) throws {
```

Comment:
Is this required for this PR? What was going on with it?

Reply:
This reminded me of something similar: https://github.com/getsentry/sentry-cocoa/pull/5059

Could we try the same fix here, prefixing as `Foundation.Data` or `Swift.Data`? I don't actually see that string anywhere already in the codebase so maybe they backed out the change but forgot to update the PR description. It's kind of hard to follow :/

Reply:
This shouldn't be here, I moved to https://github.com/getsentry/sentry-cocoa/pull/5695
Will move this PR back to draft